package.json

```diff
@@ -85,7 +85,7 @@
     "symbol-observable": "1.0.4",
     "temp": "0.8.3",
     "url-join": "1.1.0",
-    "urlregexp": "1.0.2",
+    "url-regex": "^4.0.0",
```

Comment:
We do use shrinkwrap, but we like to pin the version here anyway so that only things we are deliberately changing change when we regenerate the shrinkwrap. Please just specify "4.0.0."